wordpress-mobile/gutenberg-mobile
# Add integration tests to cover replace functionality of VideoPress block

Related to https://github.com/wordpress-mobile/gutenberg-mobile/issues/5734.

**To test:**
1. Run the command: `npm run test` and observe that all tests pass.
2. Check that all CI jobs succeed.

PR submission checklist:
